modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIPropertyRelevantObjectEntryModelListener.java

```diff
@@ -96,7 +103,36 @@ private void _scheduleAPIProperties(long apiPropertyId) {
 			});
 	}
 
-	private boolean _validate(
+	private void _validate(
+		long apiSchemaId, String objectFieldERC,
```

Comment:
I am not sure if we are simplifying, as from outside the parameters seem more complex. Why don't you get the parameters inside the method?

Reply:
As we obtain the apiSchemaId from the `Map<String, Serializable> values`, I passed it directly instead of passing the values and get the necessary fields, what do you think?